mysql/datadog_checks/mysql/const.py

```diff
@@ -235,6 +235,11 @@
     'wsrep_flow_control_sent': ('mysql.galera.wsrep_flow_control_sent', MONOTONIC),
     'wsrep_cert_deps_distance': ('mysql.galera.wsrep_cert_deps_distance', GAUGE),
     'wsrep_local_send_queue_avg': ('mysql.galera.wsrep_local_send_queue_avg', GAUGE),
+    'wsrep_replicated_bytes': ('mysql.galera.wsrep_replicated_bytes', GAUGE),
+    'wsrep_received_bytes': ('mysql.galera.wsrep_received_bytes', GAUGE),
+    'wsrep_received': ('mysql.galera.wsrep_received', GAUGE),
+    'wsrep_local_state': ('mysql.galera.wsrep_local_state', GAUGE),
+    'wsrep_local_cert_failures': ('mysql.galera.wsrep_local_cert_failures', GAUGE),
```

Comment:
I'm wondering if these 5 new metrics shouldn't be defined as `MONOTONIC`, because they are all counters of "how many XX (events or bytes) occurred since the node started".

Reply:
@jmeunier28 : what do you think of this ?

Reply:
good question I would say that makes sense for things that are events e.g.: `wsrep_local_cert_failures` should be a count

I think `wsrep_local_state` makes sense as a gauge bc its a state that fluctuates between defined numbers. As far as bytes written/received go, I think it also makes sense to just leave it as a GAUGE assuming the last number you receive from the query is a Total, which it looks like it is

Reply:
Ok thanks, I updated `wsrep_local_cert_failures` to `MONOTONIC` type and let the others as `GAUGE`